### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by GuliaShubham**
Added YouTube Java tutorials playlist link from one of the best institutes in India to learn coding.

### CODE REVIEW

**Comment by GuliaShubham** [comment]
@shahednasser please have a look I just Made the changes.

**Comment by GuliaShubham** [comment]
@shahednasser please take a look at the resources I have put on the list.

File: README.md [hunk 1]
```diff
@@ -674,7 +675,7 @@ To add a resource or contribute, please check out our contribution guide [here](
 - [MySQL Tutorial for Beginners - Programming with Mosh](https://youtu.be/7S_tz1z_5bA)
 - [SQL Tutorial - Full Database Course for Beginners - freeCodeCamp](https://youtu.be/HXV3zeQKqGY)
 - [MySQL Database - Full Course - freeCodeCamp](https://youtu.be/ER8oKX5myE0)
-
+- [MySQL tutorial](https://www.youtube.com/watch?v=u96rVINbAUI&list=PLZlA0Gpn_vH9lZBruSHgs41ldy5gZgknU)
```
Comment (shahednasser) [inline README.md:678]: This does not qualify so please remove it

File: README.md [hunk 2]
```diff
@@ -687,7 +688,7 @@ To add a resource or contribute, please check out our contribution guide [here](
 - [Deploy Node App on AWS EC2](https://jobin.hashnode.dev/deploy-your-node-js-app-into-production-with-nginx-and-ssl-installationstep-by-step-tutorial)
 - [30SecondsOfCode Node JS Snippets](https://www.30secondsofcode.org/js/t/node/p/1)
 - [The only NodeJs introduction you’ll ever need](https://codeburst.io/the-only-nodejs-introduction-youll-ever-need-d969a47ef219)
-
+- [Introduction to Node JS](https://www.youtube.com/watch?v=VShtPwEkDD0&list=PLZlA0Gpn_vH_uZs4vJMIhcinABSTUH2bY)
```
Comment (shahednasser) [inline README.md:691]: Change to the link of the playlist page

File: README.md [hunk 3]
```diff
@@ -544,7 +545,7 @@ To add a resource or contribute, please check out our contribution guide [here](
 - [Learn Java 8 - Full Tutorial for Beginners - freeCodeCamp](https://youtu.be/grEKMHGYyns)
 - [Java Beginner Course - freeCodeCamp](https://youtu.be/7WiJGTPuVeU)
 - [Java Tutorial for Beginners - Programming with Mosh](https://youtu.be/eIrMbAQSU34)
-
+- [Java Beginners to Pro Course](https://www.youtube.com/playlist?list=PL9gnSGHSqcnr_DxHsP7AW9ftq0AtAyYqJ)
```
Comment (shahednasser) [inline README.md:548]: Change the title to the title of the playlist

**Review by shahednasser — CHANGES_REQUESTED** [review]

**Comment by GuliaShubham** [comment]
@shahednasser Please check now. The git tutorial is a single video and not a playlist. I changed the title of the playlist and removed the one you asked for.

File: README.md [hunk 4]
```diff
@@ -544,7 +545,7 @@ To add a resource or contribute, please check out our contribution guide [here](
 - [Learn Java 8 - Full Tutorial for Beginners - freeCodeCamp](https://youtu.be/grEKMHGYyns)
 - [Java Beginner Course - freeCodeCamp](https://youtu.be/7WiJGTPuVeU)
 - [Java Tutorial for Beginners - Programming with Mosh](https://youtu.be/eIrMbAQSU34)
-
+- [Java + DSA + Interview Preparation Course](https://www.youtube.com/playlist?list=PL9gnSGHSqcnr_DxHsP7AW9ftq0AtAyYqJ)
```
Comment (shahednasser) [inline README.md:548]: You need to add a new line below this.

File: README.md [hunk 5]
```diff
@@ -674,7 +675,7 @@ To add a resource or contribute, please check out our contribution guide [here](
 - [MySQL Tutorial for Beginners - Programming with Mosh](https://youtu.be/7S_tz1z_5bA)
 - [SQL Tutorial - Full Database Course for Beginners - freeCodeCamp](https://youtu.be/HXV3zeQKqGY)
 - [MySQL Database - Full Course - freeCodeCamp](https://youtu.be/ER8oKX5myE0)
-
+-
```
Comment (shahednasser) [inline README.md:678]: Remove the `-` here. it should be an empty line

File: README.md [hunk 6]
```diff
@@ -687,7 +688,7 @@ To add a resource or contribute, please check out our contribution guide [here](
 - [Deploy Node App on AWS EC2](https://jobin.hashnode.dev/deploy-your-node-js-app-into-production-with-nginx-and-ssl-installationstep-by-step-tutorial)
 - [30SecondsOfCode Node JS Snippets](https://www.30secondsofcode.org/js/t/node/p/1)
 - [The only NodeJs introduction you’ll ever need](https://codeburst.io/the-only-nodejs-introduction-youll-ever-need-d969a47ef219)
-
+- [Introduction to Node.js](https://www.youtube.com/playlist?list=PLZlA0Gpn_vH_uZs4vJMIhcinABSTUH2bY)
```
Comment (shahednasser) [inline README.md:691]: Add a new empty line below this

**Review by shahednasser — CHANGES_REQUESTED** [review]
I have a few last changes below. I'll also add the `hacktoberfest-accepted` label in case this PR doesn't get merged before the end of the day

**Comment by GuliaShubham** [comment]
@shahednasser tried to resolve the problems please have a look.

**Comment by GuliaShubham** [comment]
@shahednasser please have a look made some new commits and the previous is also changed.

**Comment by shahednasser** [comment]
Great thanks!

**Pull request merged**